### PR TITLE
Fixed typos 669e81368e52b3a5c35a2dc5.md

### PR DESCRIPTION
**Pull request by Nikunj2608**
Added optimal text and fixed down typos in it

Checklist:

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #[56491](https://github.com/freeCodeCamp/freeCodeCamp/issues/56491)

### CODE REVIEW

**Comment by Nikunj2608** [comment]
This pull request includes changes to the `curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md` file to improve readability and consistency in the instructions and code examples. The most important changes include simplifying the language in the user stories, ensuring consistent formatting, and correcting minor typos.

Improvements to readability and consistency:

* Simplified the language in the user stories to make them clearer. (`curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md`)
* Ensured consistent formatting by removing backticks around HTML element names in the instructions. (`curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md`)
* Corrected a typo in the paragraph describing front-end developers. (`curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md`)

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
The file is now unusable by the build tools. Please restore all the changes that were not what is illustrated in the issue.

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 1]
```diff
@@ -27,35 +27,31 @@ You should have a `main` element inside your `body` element.
 ```js
 assert.equal(document.querySelector('body :first-child').tagName, 'MAIN');
 ```
-
-Your `main` element should be the only child of the `body` element.
+Your main element should be the only child of the body element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:30]:
```suggestion

Your main element should be the only child of the body element.
```
empty lines around code fences are needed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 2]
```diff
@@ -27,35 +27,31 @@ You should have a `main` element inside your `body` element.
...
 
 ```js
 assert.equal(document.querySelector('body').children.length, 2);
 ```
-
-You should have an `h1` element with the topic of your page inside the `main` element.
+You should have an h1 element with the topic of your page inside the main element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:35]:
```suggestion

You should have an h1 element with the topic of your page inside the main element.
```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 3]
```diff
@@ -27,35 +27,31 @@ You should have a `main` element inside your `body` element.
...
 
 ```js
 assert.isAbove(document.querySelector('h1')?.innerText.length, 0);
 ```
-
-You should have a paragraph introducing the topic of your page below your `h1` element.
+You should have a paragraph introducing the topic of your page below your h1 element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:40]:
```suggestion

You should have a paragraph introducing the topic of your page below your h1 element.
```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 4]
```diff
@@ -65,8 +61,7 @@ for (let section of sections) {
     assert.isAbove(section.children[0]?.innerText.length, 0);
 }
 ```
-
-Each `section` element should contain a `p` element to introduce the video content as its second child.
+Each section element should contain a p element to introduce the video content as its second child.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:64]:
```suggestion

Each section element should contain a p element to introduce the video content as its second child.
```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 5]
```diff
@@ -76,8 +71,7 @@ for (let section of sections) {
     assert.isAbove(section.children[1]?.innerText.length, 0);
 }
 ```
-
-Each `section` element should contain an `iframe` element as its third child.
+Each section element should contain an iframe element as its third child.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:74]:
```suggestion

Each section element should contain an iframe element as its third child.
```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 6]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
     assert.equal(section.children[2]?.tagName, 'IFRAME');
 }
 ```
-
-Your first `iframe` element should have a `src` attribute set to a valid video.
-
+Your first iframe element should have a src attribute set to a valid video.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:83]:
```suggestion

Your first iframe element should have a src attribute set to a valid video.

```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 7]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
...
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.src.length, 0)
 ```
-
-Your first `iframe` element should have a `title` attribute to describe the embedded content.
+Your first iframe element should have a title attribute to describe the embedded content.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:87]:
```suggestion

Your first iframe element should have a title attribute to describe the embedded content.
```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 8]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.title.length, 0)
 ```
-
-Your first `iframe` element should have a `height` attribute.
-
+Your first iframe element should have a height attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:92]:
```suggestion

Your first iframe element should have a height attribute.

```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 9]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
...
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.height.length, 0)
 ```
-
-Your first `iframe` element should have a `width` attribute.
-
+Your first iframe element should have a width attribute.
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
 ```
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:99]:
````suggestion

Your first iframe element should have a width attribute.

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
```
````

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 10]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
...
-Your second `iframe` element should have a `src` attribute set to a valid video.
-
+Your second iframe element should have a src attribute set to a valid video.
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.src.length, 0)
 ```
-
-Your second `iframe` element should have a `title` attribute to describe the embedded content.
-
+Your second iframe element should have a title attribute to describe the embedded content.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:104]:
```suggestion

Your second iframe element should have a title attribute to describe the embedded content.

```

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 11]
```diff
@@ -86,82 +80,59 @@ for (let section of sections) {
...
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.title.length, 0)
 ```
-
-Your second `iframe` element should have a `height` attribute.
-
+Your second iframe element should have a height attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:108]: and so on

**Review by majestic-owl448 — COMMENTED** [review]
you need to have empty lines around code blocks

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md (hunk 1 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:30]: you also removed backticks, do not do that

Comment (Nikunj2608) [reply curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:30]: @ilenia-magoni sorry for the inconvenience from my side , I am new to this

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 12]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
...
-
-Your `main` element should be the only child of the `body` element.
+Your main element should be the only child of the body element.
 
 ```js
 assert.equal(document.querySelector('body').children.length, 2);
 ```
-
-You should have an `h1` element with the topic of your page inside the `main` element.
+You should have an h1 element with the topic of your page inside the main element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:35]: still missing an empty line here, even if you marked this as resolved

**Comment by ilenia-magoni** [comment]
you also still need to fix the PR title

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 13]
```diff
@@ -9,7 +9,7 @@ demoType: onClick
 # --description--
 
 
-Fulfill the user stories below and get all the tests to pass to complete the lab.
+Fulfill the user stories below and pass all the tests to complete the lab.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:12]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 14]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
 ```js
 assert.equal(document.querySelector('body :first-child').tagName, 'MAIN');
 ```
-
-Your `main` element should be the only child of the `body` element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:31]: these two lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 15]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
...
+Your main element should be the only child of the body element.
 
 ```js
 assert.equal(document.querySelector('body').children.length, 2);
 ```
-
-You should have an `h1` element with the topic of your page inside the `main` element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:37]: these two lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 16]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
...
+You should have an h1 element with the topic of your page inside the main element.
 
 ```js
 assert.isAbove(document.querySelector('h1')?.innerText.length, 0);
 ```
-
-You should have a paragraph introducing the topic of your page below your `h1` element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:43]: these two lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 17]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
...
+You should have a paragraph introducing the topic of your page below your h1 element.
 
 ```js
 const p = document.querySelectorAll('p')[0];
 assert.equal(p?.previousElementSibling?.tagName, 'H1');
 assert.isAbove(p?.innerText.length, 0);
 ```
 
-You should have three `section` elements below your first `p` element.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:51]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 18]
```diff
@@ -66,7 +63,7 @@ for (let section of sections) {
 }
 ```
 
-Each `section` element should contain a `p` element to introduce the video content as its second child.
+Each section element should contain a p element to introduce the video content as its second child.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:66]: these lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 19]
```diff
@@ -77,7 +74,7 @@ for (let section of sections) {
 }
 ```
 
-Each `section` element should contain an `iframe` element as its third child.
+Each section element should contain an iframe element as its third child.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:77]: these lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 20]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
 }
 ```
 
-Your first `iframe` element should have a `src` attribute set to a valid video.
+Your first iframe element should have a src attribute set to a valid video.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:87]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 21]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.src.length, 0)
 ```
 
-Your first `iframe` element should have a `title` attribute to describe the embedded content.
+Your first iframe element should have a title attribute to describe the embedded content.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:93]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 22]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.title.length, 0)
 ```
 
-Your first `iframe` element should have a `height` attribute.
+Your first iframe element should have a height attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:99]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 23]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.height.length, 0)
 ```
 
-Your first `iframe` element should have a `width` attribute.
+Your first iframe element should have a width attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:105]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 24]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
 ```
 
-Your second `iframe` element should have a `src` attribute set to a valid video.
+Your second iframe element should have a src attribute set to a valid video.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:111]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 25]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.src.length, 0)
 ```
 
-Your second `iframe` element should have a `title` attribute to describe the embedded content.
+Your second iframe element should have a title attribute to describe the embedded content.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:117]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 26]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.title.length, 0)
 ```
 
-Your second `iframe` element should have a `height` attribute.
+Your second iframe element should have a height attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:123]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 27]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.height.length, 0)
 ```
 
-Your second `iframe` element should have a `width` attribute.
+Your second iframe element should have a width attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:129]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 28]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[1]?.width.length, 0)
 ```
 
-Your third `iframe` element should have a `src` attribute set to a valid video.
+Your third iframe element should have a src attribute set to a valid video.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:135]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 29]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[2]?.src.length, 0)
 ```
 
-Your third `iframe` element should have a `title` attribute to describe the embedded content.
+Your third iframe element should have a title attribute to describe the embedded content.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:141]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 30]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[2]?.title.length, 0)
 ```
 
-Your third `iframe` element should have a `height` attribute.
+Your third iframe element should have a height attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:147]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 31]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[2]?.height.length, 0)
 ```
 
-Your third `iframe` element should have a `width` attribute.
+Your third iframe element should have a width attribute.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:153]: this line should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 32]
```diff
@@ -87,81 +84,81 @@ for (let section of sections) {
...
 
 ```js
 assert.isAbove(document.querySelectorAll('iframe')[2]?.width.length, 0)
 ```
 
-# --seed--
 
-## --seed-contents--
+# --seed--
+# --seed-contents--
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:161]: these lines should not be changed

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md [hunk 33]
```diff
@@ -27,35 +27,32 @@ You should have a `main` element inside your `body` element.
...
-You should have three `section` elements below your first `p` element.
+You should have three section elements below your first p element.
 
 ```js
 assert.equal(document.querySelectorAll('section')?.length, 3);
 assert.equal(document.querySelectorAll('section')[0]?.previousElementSibling?.tagName, 'P')
 ```
 
-Each `section` element should contain an `h2` element to give a title to that section as its first child.
+Each section element should contain an h2 element to give a title to that section as its first child.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:55]:
```suggestion
Each `section` element should start with an `h2` element that serves as the title for that section.
```

you need to make the required change

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
I went line by line, this should be more clear. Also not forget the title, you will need to change that before I will review again

File: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md (hunk 3 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md:43]: you should not have changed this line, I'm unresolving the conversation

**Pull request closed without merging**